src/stories/Admin/CanDeleteAdmin/story.js

```diff
@@ -0,0 +1,66 @@
+const validator = requireValidator();
+const adminRepo = requireRepo("admin");
+const tokensRepo = requireRepo("tokens");
+const findKeysFromRequest = requireUtil("findKeysFromRequest");
+
+const prepare = ({ reqQuery, reqBody, reqParams, req }) => {
+  const payload = findKeysFromRequest(req, ["email", "password"]);
+  payload["jti"] = req.jti;
+  return payload;
+};
+
+const authorize = async ({ prepareResult }) => {
+  let unauthorizedObject = {
+    statusCode: 401,
+    message: "Unauthorized",
+  };
+
+  try {
+    if (prepareResult.jti !== undefined) {
+      let token = await tokensRepo.first({
+        uuid: prepareResult.jti,
+      });
+      if (token["issuer"] === "admin") {
+        let isSuperUser = await adminRepo.isSuperUser(token.sub);
+        if (isSuperUser) {
+          return token;
+        } else {
+          throw unauthorizedObject;
+        }
+      } else {
+        throw unauthorizedObject;
+      }
+    } else {
+      throw unauthorizedObject;
+    }
+  } catch (error) {
+    throw error;
+  }
+};
+
+const handle = async ({ prepareResult, authorizeResult }) => {
+  try {
+    let payload = {
+      email: prepareResult.email,
+      password: prepareResult.password,
+    };
+    return await adminRepo.createAdmin(payload);
```

Comment:
@praveen-betalectic The story is canDeleteAdmin, but we are calling create Admin right?

Reply:
@rjvim code changed please review again.

Reply:
@praveen-betalectic There is a postman collection in repo; please import that into your Postman add new endpoints also and update it back into repo.

Reply:
@rjvim postman collection updated.